settings/blt.settings.php

```diff
@@ -65,7 +65,9 @@
 $is_acsf_env = (!empty($ah_group) && file_exists("/mnt/files/$ah_group.$ah_env/files-private/sites.json"));
 // @todo Maybe check for acsf-tools.
 $is_acsf_inited = file_exists(DRUPAL_ROOT . "/sites/g");
-$acsf_db_name = $is_acsf_env ? $GLOBALS['gardens_site_settings']['conf']['acsf_db_name'] : NULL;
+if(isset($GLOBALS['gardens_site_settings'])) {
+  $acsf_db_name = $is_acsf ? $GLOBALS['gardens_site_settings']['conf']['acsf_db_name'] : NULL;
```

Comment:
It's a little too dangerous to unset this variable, which may be used by customer's scripts.

$acsf_db_name should always be set, but it should be null if $GLOBALS['gardens_site_settings'] is not set.

We can just modify the logic in the ternary condition. 

```
$acsf_db_name = isset($GLOBALS['gardens_site_settings']) && $is_acsf ? $GLOBALS['gardens_site_settings']['conf']['acsf_db_name'] : NULL;
```